frontend/src/components/appraisal.jsx

```diff
@@ -268,7 +268,7 @@ const Appraisal = () => {
                       <option value="IDOTRR">Iowa DOT and Rail Road</option>
                       <option value="MeadowV">Meadow Village</option>
                       <option value="Mitchel">Mitchell</option>
-                      <option value="Names">North Ames</option>
+                      <option value="NAmes">North Ames</option>
```

Comment:
This should be '"Names"', single quotes enclosing it since double quotes are used in the data.

Reply:
i mean '"NAmes"'